src/layouts/SideBar/SideBar.component.tsx

```
@@ -1,8 +1,53 @@
+            >
+                {sideBarBottomIcons &&
+                    sideBarBottomIcons.map((icon, index) => {
+                        const IconComponent =
+                            iconMap[icon as keyof typeof iconMap];
+                        if (!IconComponent) return null;
+                        return (
+                            <IconContainer key={index}>
+                                <IconComponent />
+                            </IconContainer>
+                        );
+                    })}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)